relay-dynamic-config/src/feature.rs

```diff
@@ -21,6 +21,11 @@ pub enum Feature {
     /// Serialized as `organizations:session-replay-combined-envelope-items`.
     #[serde(rename = "organizations:session-replay-combined-envelope-items")]
     SessionReplayCombinedEnvelopeItems,
+    /// Disables select organizations from processing mobile replay events.
+    ///
+    /// Serialized as `organizations:session-replay-video-denylist`.
+    #[serde(rename = "organizations:session-replay-video-denylist")]
```

Comment:
Just a nit, but I would not mention a denylist here, because from Relay's point of view that's an implementation detail of the sentry-side feature handler.

```suggestion
    #[serde(rename = "organizations:session-replay-video-disabled")]
```